pytx/pytx/malware_family.py

```diff
@@ -17,8 +17,10 @@ class MalwareFamily(Common):
         mf.FAMILY_TYPE,
         mf.MALICIOUS,
         mf.NAME,
+        mf.PRIVACY_TYPE,
         mf.SAMPLE_COUNT,
-        mf.SHARE_LEVEL
+        mf.SHARE_LEVEL,
+        mf.SUBMITTER_COUNT
```

Comment:
Please add a trailing comma so that we can preserve blame. We haven't really modified the line above, but it's included in this commit because we're adding the trailing comma now.